SWLOR.Game.Server/Service/PerkService/PerkBuilder.cs

```diff
@@ -160,6 +160,20 @@ public PerkBuilder RequirementUnlocked()
             return this;
         }
 
+        /// <summary>
+        /// Adds a requirement that the player must have leveled a specific other perk.
+        /// </summary>
+        /// <param name="mustHavePerkType">The type of perk the player cannot have.</param>
```

Comment:
Think these comments were copy/pasted but not updated to reflect the purpose of this method. Can you please correct them?